src/repositories/mongo_config.py

```diff
@@ -2,4 +2,5 @@
 
 MONGO_CONNECTION_STRING = "DB-CONNECTION"
 DATABASE_NAME = "inventory"
-MONGO_CERT = ssl.CERT_NONE
\ No newline at end of file
+MONGO_CERT = ssl.CERT_NONE
+HEADER = {"Access-Control-Allow-Origin": "*"}
```

Comment:
This constant header is part of the controller config. but no big deal

Reply:
Didn't quite get this part.. Could you be more specific?

Reply:
Got it, will fix